import-scripts/import_scripts/channel.py

```diff
@@ -462,7 +462,7 @@ def gen():
 
             position = data["meta"].get("position")
             if position and position.startswith("/nix/store"):
-                position = position[44:]
+                position = position.split("/", 4)[-1]
```

Comment:
I think some time ago this was actually working. but then something changed in nixpkgs and `source/` was added. It think we need to be a little more defensive here. I would only remove `source/` and nothing else. We definetly dont want to rely on structure here.

```
position = position[44:]
if position.startswith('source/'):
    position = position[len('source/'):]
```

or something similar

Reply:
But then things will break again if `source/` changes to `src/` or whatnot. I think we can reasonably rely on the fact that it will always look like `/nix/store/hash-name/actual_path_in_nixpkgs`, and therefore stripping the first three path components should be enough.